app/views/dashboard/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1><%= current_user.name %></h1>
+<body id="user-header">
```

Comment:
There should only be a single body tag in an HTML document, and it's already defined in `app/views/layouts/application.html.erb`